### DISCUSSION TITLE
docker pull timeout

### DISCUSSION THREAD

**Discussion by liuxinwang** [hhyo/Archery#2680] [General]
现在就是archery docker pull timeout，修改源后其它都可以正常pull
error pulling image configuration: download failed after attempts=6: dial tcp 162.125.32.6:443: i/o timeout

cat /etc/docker/daemon.json 
{
    "registry-mirrors": [
        "https://dockerproxy.com",
        "https://docker.mirrors.ustc.edu.cn",
        "https://docker.nju.edu.cn",
        "https://mirror.baidubce.com"
    ]
}

**Comment by LeoQuote** [comment]
非 archery问题，请自行排查解决，欢迎提出解决方案

**Comment by liuxinwang** [comment]
临时替代镜像
https://github.com/DaoCloud/public-image-mirror/issues/4253?notification_referrer_id=NT_kwDOATFoSbQxMTAzNTk3ODkxNzoyMDAxNTE3Nw#event-13111036208

**Comment by LookUpPls** [comment]
所有国内镜像地址被关掉了
